.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: dart
+sudo: false
+dart:
+  - dev
+  - stable
+  - 1.24.2
+  - 1.23.0
+  - 1.22.1
+
+dart_task:
```

Comment:
That's true, but it also decreases throughput and soaks up builders that other packages may need.